Common/AnalysisTools.h

```diff
@@ -68,6 +68,36 @@ enum class GenLeptonMatch : int {
   NoMatch = 6
 };
 
+
+enum class WorkingPointsTauVSmu : int {
```

Comment:
reuse these definitions in baseline. E.g. in initialise:
```python
...
globals()["WorkingPointsTauVSmu"] = ROOT.WorkingPointsTauVSmu
...
```

Reply:
I changed the class definition inserting it in Utilities.py